apps/cms/src/payload.config.ts

```diff
@@ -0,0 +1,28 @@
+import path from 'path';
+
+import { viteBundler } from '@payloadcms/bundler-vite';
+import { mongooseAdapter } from '@payloadcms/db-mongodb';
+import { payloadCloud } from '@payloadcms/plugin-cloud';
+import { slateEditor } from '@payloadcms/richtext-slate';
+import { buildConfig } from 'payload/config';
+
+import Users from './collections/Users';
+
+export default buildConfig({
+  admin: {
+    user: Users.slug,
+    bundler: viteBundler(),
+  },
+  editor: slateEditor({}),
```

Comment:
Looking at the Payload docs I think it might be worthwhile to use their beta Lexical editor--it has a much simpler API/system to generate HTML from the editor's JSON.